README.md

```diff
@@ -130,6 +130,8 @@ This file must be located in the project root and declares the project settings.
 | assets                    |                             | static asset files, that should be copied to the styleguide directory, head for "Assets" for further information
 | content                   |                             | list of page configurations, take a look at "Content"
 | componentDocs             | "."                         | path of component doc files, relative to the given component.yml. "doc" e.g says "all documents have to be placed in a sub-directory doc adjacent to the component.yml"
+| assetHost                 |                             | url origin for external assets
+| themes                    |                             | list of style themes, of that each defines a different themed version of the components
```

Comment:
I renamed `themes` to `componentThemes` as you suggested. Inside the `layoutContext` it is still called `themes` since this is how it is used inside the stylegen-theme template, so we would need to change it there too. 

I think that would probably be the better solution, so the naming is consistent. But it would need another PR of the stylegen-flatwhite-theme.